app/assets/javascripts/assignments.js

```diff
@@ -25,7 +25,11 @@ $(document).ready(function() {
                             klass = 'btn-success';
                             break;
                     }
-                    $div.html('<a href="' + actions_map[mid].url + '" class="btn ' + klass + '" >' + actions_map[mid].action + '</a>')
+                    if ( actions_map[mid].action == 'Attempt' ){
+                        $div.html('<a href="' + actions_map[mid].url + '" class="btn ' + klass + '" data-disable-with="Attempting..." >' + actions_map[mid].action + '</a>');
+                    } else {
+                        $div.html('<a href="' + actions_map[mid].url + '" class="btn ' + klass + '" >' + actions_map[mid].action + '</a>');
```

Comment:
Can we do

    $('<a>', { href: 'actionsMap[mid.url]',
               class: 'brn ' + klass,
                   ... })